project_zoo/animal.py

```diff
@@ -0,0 +1,29 @@
+# Calvin Hsieh chgenedu@gmail.com
+# oa-professional-software-development-bootcamp
+# Week 2
+# Nov 5, 2023
+#
+# animal.py
+
+from abc import abstractmethod
+
+class Animal:
+	def __init__(self, name, age):
+		self.__name = name
```

Comment:
nit: I would use the private variables with a single underscore here. 

For an explanation: https://stackoverflow.com/a/1301369

Reply:
I used double underscore because I was following Leon's week 2 lecture example where he used double underscore for private variables. (He also said double underscore is for private variables and single underscore is for protected variables.) Why would you use a single underscore in this situation? On the stackoverflow page, people don't seem to agree on the use of single and double underscore.

Reply:
yeah as i mentioned in the session, pythons class member visibility is kinda wonky, because it's not fully enforced and the community is a bit torn. I wouldn't pin down to use the underscore semantics, since you won't find it in most of the public python code.

Reply:
I see, Leon. Thanks for the comment!